pkg/system/lcow_unsupported.go

```diff
@@ -1,27 +1,14 @@
-// +build !windows windows,no_lcow
-
 package system // import "github.com/docker/docker/pkg/system"
 import (
 	"runtime"
 	"strings"
-
-	specs "github.com/opencontainers/image-spec/specs-go/v1"
 )
 
-// InitLCOW does nothing since LCOW is a windows only feature
-func InitLCOW(_ bool) {}
-
 // LCOWSupported returns true if Linux containers on Windows are supported.
 func LCOWSupported() bool {
 	return false
 }
```

Comment:
Is this still used somewhere, or should this whole file likely be going away?

Reply:
Let me double check if it's still called in this diff. Once everything is done, it should definitely be gone.

I mostly opened this PR (smaller subset of https://github.com/moby/moby/pull/42170) for my own sanity, as the LCOW code is in so many places, that "chipping it off" in smaller bits made it somewhat easier both for me, as well as for reviewers. 😓

Reply:
Just checked, and `LCOWSupported()` is still used in various places (but now of course always `false`)

Reply:
Sounds good! 👍